f5lbaasdriver/v2/bigip/agent_rpc.py

```diff
@@ -272,3 +272,77 @@ def delete_health_monitor(self, context, health_monitor, service, host):
                 service=service
             ),
             topic=topic)
+
+    @log_helpers.log_method_call
```

Comment:
In relation to our offline conversation, the driver_v2 module needs to match calls expected here. We also need to ensure what the driver_v2 module gets as input to its CRUD operations.